src/sim/controller/functions/v_ping.py

```diff
@@ -0,0 +1,5 @@
+# Send ICMP Ping out of Single Transceiver Directed at a Single Robot
+# Used for Associate (In Node Discovery) and Reassociate (in New Visible)
+def v_associate(robotIP, vg):
```

Comment:
lets be consistent with naming. pick a way. I like 'v_' for threads like you have here